src/components/Editor.jsx

```diff
@@ -15,6 +15,8 @@ class Editor extends React.Component {
       this._editor.setValue(nextProps.source);
     }
 
+    //this disables autoclosing of HTML tags
+    this._editor.setBehavioursEnabled(false);
```

Comment:
I think we want this in the `_setupEditor` method (which is where the `_editor` property is initialized); as written, this will re-apply the setting each time the source is updated, which doesn’t seem necessary.